cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -276,7 +269,11 @@ def certificates_detail_handler(request, course_key_string, certificate_id):
     store = modulestore()
     with store.bulk_operations(course_key):
         course = get_course_and_check_access(course_key, request.user)
-        certificates_list = course.certificates['certificates']
+        certificates_list = []
+        if course.certificates.get('certificates') is not None:
+            certificates_list = course.certificates['certificates']
+        else:
+            course.certificates['certificates'] = []
 
```

Comment:
just for safe side i have added this check. i know in edit certificate scenario , we have already a poplutted dict under course.certificates.